MetaGenerator/FileGenerator.cs

```diff
@@ -33,11 +33,14 @@ internal class FileGenerator : IFileGenerator
         public Task FromTemplate(string templatename, string destinationPath)
             => FromTemplate(templatename, destinationPath, Encoding.UTF8);
 
-        public async Task FromTemplate(string templateName, string destinationPath, Encoding encoding)
+        public Task FromTemplate(string templateName, string destinationPath, Encoding encoding)
+            => FromTemplate(templateName, _templateLoader.GetTemplate(templateName), destinationPath, encoding);
+
+        public async Task FromTemplate(string templateName, string templateContent, string destinationPath, Encoding encoding)
```

Comment:
Where is this new overload used?

Reply:
Addressed in: [Make overload method private](https://github.com/messerli-informatik-ag/meta-generator/pull/22/commits/a7ba6a30850c58886d50bb61ef86ac08a6788eed)